openerp/addons/base/ir/ir_qweb.py

```diff
@@ -222,7 +222,7 @@ def eval_format(self, expr, qwebcontext):
             return str(expr % qwebcontext)
         except Exception:
             template = qwebcontext.get('__template__')
-            raise_qweb_exception(message="Format error for expression %r" % expr, expression=expr, template=template)
+            raise_qweb_exception(message="Format error for expression %r" % expr, expression=expr, template_attributes=template_attributes)
```

Comment:
Since the underlying exception has the template_name passing in the attributes to better identify which node has a rendering issue.Otherwise you would just know that a <span>/<td> element has a rendering error with no exact location or distinction criteria